apps/percona/chaos/node_freeze/run_litmus_test.yml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: node-freez-
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        freezer: node-mem
+
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: vvibhr995/ansible-aws:cha1
```

Comment:
image ?

Reply:
@ksatchit Once i am finished with testing i am going to change it